pkgs/top-level/pkg-config-packages.nix

```diff
@@ -0,0 +1,202 @@
+/* A set of aliases to be used in generated expressions.
+
+   In case of ambiguity, this will pick a sensible default.
```

Comment:
I think we should provide all packages that match, something like one of:
```nix
pkgConfigName = [ pkg0 pkg1 ];
pkgConfigName = { inherit pkg0 pkg1; };
```
A key idea of pkg-config is that it tracks *interfaces* not implementations.

Reply:
My intent is for this to be a slightly curated set of packages, providing defaults for lang2nix integrations. Adding multiple versions / variants / ... here will add a lot more churn and will confuse lang2nix integrations that don't have a means to pick one. I feel like this would be more suitable for a version 2.

How about I call this package set `defaultPkgConfigPackages` instead? That will leave room for a fancier implementation with a nice name.

Reply:
@roberth sounds good!

Reply:
- https://github.com/NixOS/nixpkgs/issues/212290

Reply:
The list data is in a `.json` now that can be evolved if needed. It's already quite extensible because it uses records wherever it can; top-level and in each module. It also has a top-level version field just in case.